test/OpenTelemetry.Api.Tests/Trace/TracerTest.cs

```diff
@@ -283,6 +283,32 @@ public void CreateSpan_NotSampled()
         Assert.False(span.IsRecording);
     }
 
+    [Fact]
+    public void Tracer_Disposed_Becomes_Noop()
```

Comment:
Do we really want to use capitalized snake case in C#? (I know some existing test cases use that... I guess we want to fix them instead of follow them)

Reply:
Poor snakes they never catch a break! Updated with a new name.